docs/src/manual/grid.md

```diff
@@ -43,7 +43,13 @@ FerriteMeshParser.get_ferrite_grid
 ```
 
 If you are missing the translation of an Abaqus element that is equivalent to a `Ferrite.Cell`,
-consider to open an [issue](https://github.com/Ferrite-FEM/FerriteMeshParser.jl/issues/new) or a pull request. 
+consider to open an [issue](https://github.com/Ferrite-FEM/FerriteMeshParser.jl/issues/new) or a pull request.
+
+!!! note "Common pitfalls for reading a custom mesh"
+    In case you write your own custom mesh reader, take care of the Ferrite.jl's anti-clockwise cell node ordering.
```

Comment:
```suggestion
    In case you read custom cells or write your own custom mesh reader, take care of the Ferrite.jl's anti-clockwise cell node ordering.
```